### PR TITLE
Fix: Do not clean more levels than needed

### PR DESCRIPTION
**Pull request by localheinz**
This PR
- [x] cleans only as many levels as needed
### Before

```
$ vendor/bin/phpunit
PHPUnit 4.8.6 by Sebastian Bergmann and contributors.

...............................................................  63 / 130 ( 48%)
.....................................................R..RR..... 126 / 130 ( 96%)
R...

Time: 2.18 seconds, Memory: 12.75Mb

OK, but incomplete, skipped, or risky tests!
Tests: 130, Assertions: 203, Risky: 4.

Generating code coverage report in Clover XML format ... done

Generating code coverage report in HTML format ... done
```
### After

```
$ vendor/bin/phpunit
PHPUnit 4.8.6 by Sebastian Bergmann and contributors.

...............................................................  63 / 130 ( 48%)
............................................................... 126 / 130 ( 96%)
....

Time: 2.28 seconds, Memory: 12.50Mb

OK (130 tests, 203 assertions)

Generating code coverage report in Clover XML format ... done

Generating code coverage report in HTML format ... done
```

Follows https://github.com/thephpleague/plates/pull/78.

### CODE REVIEW

**Comment by frankdejonge** [comment]
@localheinz won't this hold the level even if at a later time an additional render is called when the level may or may not be the same?

**Comment by localheinz** [comment]
@frankdejonge 

You probably know better, what would you suggest?

I only noticed the error when running the tests, and wondered.

**Comment by reinink** [comment]
Yeah, I'll need to do some testing on this one. It makes me the most concerned, granted your solution may be good.

File: src/Template/Template.php [hunk 1]
```diff
@@ -105,6 +105,12 @@ public function path()
      */
     public function render(array $data = array())
     {
+        static $startLevel;
+
+        if ($startLevel === null) {
+            $startLevel = ob_get_level();
+        }
+
```
Comment (zreed) [inline src/Template/Template.php]: This should just be:
`$startLevel = ob_get_level();`

**Comment by reinink** [comment]
Okay I've made these improvements. You can find these changes in this commit: https://github.com/thephpleague/plates/commit/46778e8c5748fc2bc95887754c2a05d71b90fbae

I'm going to close this PR, but it's worth noting that @localheinz totally had the solution here. However, because of merge conflicts it was easier to make this commit myself. Thanks for contributing! 😎

**Comment by localheinz** [comment]
Thank you, @reinink!

**Pull request closed without merging**